hack/update-protobuf.sh

```
@@ -2,50 +2,9 @@
-# We need to make sure that protoc-gen-gogo is on the path for go-to-protobuf to run correctly.
-protoc_bin_dir=$(dirname "${PROTOC_GEN_GOGO}")
-
-PATH="$PATH:${protoc_bin_dir}" "${GO_TO_PROTOBUF}" \
-  --output-dir="${GOPATH}/src" \
-  --apimachinery-packages='-k8s.io/apimachinery/pkg/util/intstr,-k8s.io/apimachinery/pkg/api/resource,-k8s.io/apimachinery/pkg/runtime/schema,-k8s.io/apimachinery/pkg/runtime,-k8s.io/apimachinery/pkg/apis/meta/v1,-k8s.io/apimachinery/pkg/apis/meta/v1beta1,-k8s.io/api/core/v1,-k8s.io/api/rbac/v1' \
-  --go-header-file=${SCRIPT_ROOT}/hack/empty.txt \
-  --proto-import=${SCRIPT_ROOT}/third_party/protobuf \
-  --proto-import=${SCRIPT_ROOT}/vendor \
-  --proto-import=${SCRIPT_ROOT}/tools/vendor \
-  --packages="${API_PACKAGES}"
+GENERATOR=go-to-protobuf ${SCRIPT_ROOT}/hack/update-codegen.sh
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)